tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -158,9 +158,30 @@ pub trait Subscriber: 'static {
 
     /// Record a set of values on a span.
     ///
+    /// This method will be invoked when value is recorded on a span.
+    /// Recording multiple values for the same field is possible,
+    /// but the actual behaviour is implementation defined by the subscriber.
+    ///
+    /// Keep in mind that a span might not provide values for every field it declares.
```

Comment:
TIOLI `keep in mind that a span may not provide a value for each fieled it declares.`

Reply:
I've settled in between, "may" not sounds like it is forbidden